code/game/objects/structures/supplypod.dm

```diff
@@ -44,6 +44,7 @@
 	pixel_x = -16
 	pixel_y = -5
 	layer = TABLE_LAYER
+	closet_flags = CLOSET_ALLOW_OBJS | CLOSET_ALLOW_DENSE_OBJ
```

Comment:
```suggestion
	closet_flags = CLOSET_ALLOW_OBJS|CLOSET_ALLOW_DENSE_OBJ
```
From the contributing guidelines:
Operators that should not be separated by spaces:
    Bitwise OR |